include/mpix/op_correction.h

```diff
@@ -77,6 +77,8 @@ enum mpix_correction_type {
 	MPIX_CORRECTION_WHITE_BALANCE,
 	/** Apply gamma correction to non-linearly increase the brightness of the image */
 	MPIX_CORRECTION_GAMMA,
+	/** Apply color correction to every pixel*/
+	MPIX_CORRECTION_COLOR_CORRECTION,
```

Comment:
This has `CORRECTION` twice, how about matching the struct name 1:1 for consistency?

```suggestion
	/** Apply a color correction matrix (CCM) to every pixel*/
	MPIX_CORRECTION_COLOR_MATRIX
```